### DISCUSSION TITLE
Change default shell when clicking the "+" button

### DISCUSSION THREAD

**Discussion by mikefoley** [Eugeny/tabby#6407] [Q&A] [answered]
Hi,
On my Mac when I click the + button I get a csh shell. I'd rather get a zsh shell. Is there a way to change this? 
thanks.

**Comment by Eugeny** [comment, marked as answer]
Yes - Settings -> Profiles -> Default profile for new tabs.

**Reply by mikefoley** [reply]
Excellent, thanks!!

**Reply by lawweiliang** [reply]
Take notes, 
if you are using black background color like I do, it might be slightly difficult for you to see the defaul profile drop-down box. https://github.com/Eugeny/tabby/discussions/8151

**Reply by ghost** [reply]
I am using Tabby on Ubuntu. And I find the user-default shell cannot be changed, which is `/usr/bin`.

**Comment by lawweiliang** [comment]
@mikefoley 
Please close the discussion if the question was answered. 
It would be helpful to make the discussion section clean. 

Click `mark as answered` and `close the discussion`. 😄 

![image](https://user-images.githubusercontent.com/34936929/228098333-222ac3d4-4c39-4c26-a2a8-52906647e00e.png)
